rules/cloud/aws/aws_iam_s3browser_templated_s3_bucket_policy_creation.yml

```diff
@@ -0,0 +1,33 @@
+title: AWS IAM S3Browser Templated S3 Bucket Policy Creation
+id: db014773-7375-4f4e-b83b-133337c0ffee
+status: experimental
+description: Detects S3 Browser utility creating Inline IAM Policy containing default S3 bucket name placeholder value of <YOUR-BUCKET-NAME>.
+references:
+    - https://permiso.io/blog/s/unmasking-guivil-new-cloud-threat-actor
+author: daniel.bohannon@permiso.io (@danielhbohannon)
+date: 2023/05/17
+modified: 2023/05/17
+tags:
+    - attack.execution
+    - attack.t1059.009
+    - attack.persistence
+    - attack.t1078.004
+logsource:
+    product: aws
+    service: cloudtrail
+detection:
+    selection_source:
+        eventSource: iam.amazonaws.com
+        eventName: PutUserPolicy
+    filter_tooling:
+        userAgent|contains: 'S3 Browser'
+    filter_policy_resource:
+        requestParameters|contains: '"arn:aws:s3:::<YOUR-BUCKET-NAME>/*"'
```

Comment:
I think you meant this `<YOUR-BUCKET-NAME>` as a place holder? If so then you can either use a wildcard directly or we can transfer this rule to a placeholder rule.